.gitignore

```diff
@@ -24,6 +24,9 @@ _testmain.go
 *.test
 *.prof
 
+#IDE ignore
+.idea
+
```

Comment:
We usually recommend to put these developer-specific files in your global .gitignore: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/
Please remove this change.

Reply:
Removed